src/Smalot/PdfParser/Parser.php

```diff
@@ -101,7 +101,7 @@ public function parseContent(string $content): Document
         // Create structure from raw data.
         list($xref, $data) = $this->rawDataParser->parseData($content);
 
-        if (isset($xref['trailer']['encrypt'])) {
+        if (isset($xref['trailer']['encrypt']) && !$this->config->getIgnoreEncryption()) {
```

Comment:
The following change makes the code more readable in my opinion.

```suggestion
        if (isset($xref['trailer']['encrypt']) && false === $this->config->getIgnoreEncryption()) {
```